#!/usr/bin/env python
PACKAGE = "viscon"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("scale_factor", double_t, 0,"Scale factor for orb slam position",    0, 0,   10)

for i in ['x','y','z','w']:
    for j in ['p','i','d']:
        gen.add(str(j)+"_"+str(i), double_t, 0, str(j)+" parameter for PID vel control over "+str(i)+" axis",    0, -1,   1)
exit(gen.generate(PACKAGE, "simple_control.py", "Controller"))
